drivers/usb/udc/udc_it82xx2.c

```diff
@@ -1394,7 +1404,11 @@ static void suspended_handler(struct k_work *item)
 	}
 
 	it82xx2_enable_resume_int(dev, true);
+
+#if (!defined(USB_DEVICE_CONFIG_SOF_NOTIFICATIONS) || (USB_DEVICE_CONFIG_SOF_NOTIFICATIONS == 0U))
```

Comment:
There should not be any `#if (!defined(USB_DEVICE_CONFIG_SOF_NOTIFICATIONS) || (USB_DEVICE_CONFIG_SOF_NOTIFICATIONS == 0U))`
```c
	if (USB_DEVICE_CONFIG_SOF_NOTIFICATIONS) {
		it82xx2_enable_sof_int(dev, true);
	}
```

Reply:
Updated.